crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -118,18 +121,68 @@ impl Plugin for VisibilityPlugin {
     }
 }
 
+/// Calculates and updates [`Aabb`]s for [`Entities`](Entity) with [`Mesh`]es.
+/// To opt out of bound calculation for an `Entity`, give it the [`NoFrustumCulling`] component.
+///
+/// # Examples
+///
+/// To automatically get calculated `Aabb`s that will update with `Mesh` assignment/mutation, add
+/// this as a [`System`] to your `App`:
+///
+/// ```
+/// # use bevy_app::prelude::App;
+/// # use bevy_render::view::calculate_bounds;
+///
+/// App::new()
+///     .add_system(calculate_bounds)
```

Comment:
Do we normally do this kind of example for a core system that is part of the default systems that are enabled? It seems a bit overboard to me as this is how adding any system to a bevy app works.

Reply:
Oh I didn't know this was a default system! After reading the original issue I was a bit interested in how users were supposed to discover this. If it's a default system then I'd also vote to remove this example (though I'll defer to anyone else here).

Reply:
Yep, I don't think describing how to add default systems like this is useful.

Reply:
Removed in 9730917d7a5ea6082b8966f5f6e3dd12724d7317